src/core/DelayedFunctionScheduler.js

```diff
@@ -87,6 +87,42 @@ getJasmineRequireObj().DelayedFunctionScheduler = function(j$) {
       }
     };
 
+    /**
```

Comment:
Please remove the jsdoc comments from everything that isn't meant to be part of the public interface. Explanatory comments on private methods etc are fine as long as you avoid jsdoc syntax. All jsdoc comments semi-automatically become part of the published API reference documentation.

Reply:
For these two, I actually think they need to be in the public interface, right? Assuming developers can implement their own `DelayedFunctionScheduler` independently from the `Clock` implementation, these functions do need to be available to the `Clock` in order to use the auto ticking behavior.

Reply:
Users can't provide their own `DelayedFunctionScheduler`. (Well, it's technically possible to construct your own `Clock`, but [the documentation explicitly says not to](https://jasmine.github.io/api/5.3/Clock.html)). Even if it didn't, `DelayedFunctionScheduler` is an undocumented private API and while Jasmine doesn't go out of our way to block users from accessing it, we also don't put any effort into preserving compatibility with user code that does.

With a couple of edge-case exceptions, anything not mentioned on https://jasmine.github.io is a private API. Promoting private APIs to public is something I don't do without a compelling reason, because it limits our ability to evolve Jasmine. You're benefiting from that policy right now: If `DelayedFunctionScheduler` and the `Clock` constructor were already public, I'd have to decline this PR because it would break compatibility with user code that supplies its own `DelayedFunctionScheduler`.